src/MiniCover/Commands/Options/IMiniCoverOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+
+namespace MiniCover.Commands.Options
+{
+    internal interface IMiniCoverOption
+    {
+        string OptionTemplate { get; }
+        string Description { get; }
+        CommandOptionType Type { get; }
+
+        void Invalidate();
```

Comment:
This method actually validates the argument. Can you please rename it to Validate?